orc8r/cloud/helm/orc8r/values.yaml

```diff
@@ -84,20 +86,23 @@ controller:
     portStart: 9079
     portEnd: 9108
 
-  # proxy image
+  # controller image
   image:
     repository:
     tag: latest
     pullPolicy: IfNotPresent
 
   spec:
-    # Postgres configuration
-    postgres:
-      db: magma
-      host: postgresql
-      port: 5432
-      user: postgres
-      pass: postgres
+    # Postgres/mysql configuration
+    database:
+      driver: mysql      # mysql/postgres
```

Comment:
May I suggest keeping the default as postgres since all the current deployments and documentation rely on postgres? We can override this in the vals.yaml for mysql.

Reply:
fixed